test/parallel/test-webcrypto-encrypt-decrypt-rsa.js

```diff
@@ -29,13 +29,12 @@ async function importVectorKey(
   return { publicKey, privateKey };
 }
 
-async function testDecryption({
-  ciphertext,
-  algorithm,
-  plaintext,
-  hash,
-  publicKeyBuffer,
-  privateKeyBuffer }) {
+async function testDecryption({ ciphertext,
+                                algorithm,
+                                plaintext,
+                                hash,
+                                publicKeyBuffer,
+                                privateKeyBuffer }) {
```

Comment:
```suggestion
async function testDecryption({
  ciphertext,
  algorithm,
  plaintext,
  hash,
  publicKeyBuffer,
  privateKeyBuffer,
}) {
```

Reply:
For function declarations, I personally prefer the larger indentation. It seems visually more clear to me what's going on.

Reply:
For function declarations, I personally prefer the larger indentation. It seems visually more clear to me what's going on.